cmd/cloudshell_open/main.go

```diff
@@ -111,11 +112,20 @@ func run(c *cli.Context) error {
 		return err
 	}
 
+	if len(projects) == 0 {
+		fmt.Printf("%s %s Your Google Cloud Platform account has no projects!\n"+
+			"Create a new GCP project at %s\n"+
+			"and refresh this window to continue deploying this application.\n",
```

Comment:
Unfortunately refreshing doesn't seem to restart the process.

Reply:
I think I'll add a for loop with "Press enter to try again" until we get a project.

Reply:
I decided to add the "Please create a project and press enter" in a separate PR.